### ISSUE TITLE
Dual contouring

### ISSUE THREAD

**Issue by mikolalysenko** [stackgl/contributing#10]

**Comment by wwwtyro** [comment]
Is this issue addressed by http://stack.gl/packages/#mikolalysenko/surface-nets and http://stack.gl/packages/#mikolalysenko/isosurface? (thanks for these by the way, they're awesome)

**Comment by Erkaman** [comment]
Nope, he's referring to the dual-contouring technique described in [this paper](http://www.frankpetterson.com/publications/dualcontour/dualcontour.pdf), which is a technique to extract a contour(mesh) from hermite data. `isosurface` implements some techniques for extracting an isosurface from a potential-function. That's not the same thing as dual-contouring.

**Comment by mikolalysenko** [comment]
Yeah, though it is closely related to dual contouring.  The main thing that dual contouring needs is gradients (aka hermite data).  This can be done, but it would require changing the interfaces around a bit.

**Comment by wwwtyro** [comment]
Ah okay, got it. Thanks for the clarification, guys.
